### ISSUE TITLE
Custom polling intervals / automatic delays

### ISSUE THREAD

**Issue by Alwaysin** [FreshRSS/FreshRSS#968]
I have different types of feeds posting at different intervals. For example, some are posting dozens every day, others one every few days, others one every few months,...

Thus, I think we should be able to say "this feed, do not update it every hour", but the configuration can be a bit heavy.

So : we already have the statistics module, we could base on that to define automatic retrieval frequency.
Feed posting lots of articles every day ? -> Poll every hour
Feed posting every few weeks ? -> Poll every day

Something like that :) It would make the retrieval much faster, by not wasting resources every hours on feeds having something every once in a while!

Inoreader is working like that. Their description (may be better explained :)) : "The default update interval for new feeds is 1 hour. After some time (usually a day or two) the feed history is examined and it is assigned a proper interval according to some factors."

**Comment by Alkarex** [comment]
Yes, good idea. Reference https://github.com/FreshRSS/FreshRSS/issues/805#issuecomment-77449626

**Comment by ghost** [comment]
+1 for this future feature, by the way is it possible today to disable a refresh for a category by altering manually the database ?

**Comment by Alkarex** [comment]
Remember to look at the HTTP caching policy defined by the feed.
http://genma.free.fr/?RSS-Message-de-service-et-conseils-pour-l-agregateur-RSS#forum6295
